draft-ietf-tls-tls13.md

```diff
@@ -2261,13 +2261,15 @@ The "extension_data" field of this extension contains an
 
 %%% Key Exchange Messages
 
+       struct {} None;
+
        struct {
            select (Handshake.msg_type) {
                case client_hello:
                    uint32 obfuscated_ticket_age;
 
                case server_hello:
-                   struct {};
+                   None;
```

Comment:
But "{}" is not defined in Section 3. In my opinion, such an ill-defined notation is much confusing than this indirection. I would vote for `None` (or `Empty`) even if it is redundant.